samples/sample_hevc_fei/src/hevc_fei_encode.cpp

```diff
@@ -197,6 +229,45 @@ mfxStatus FEI_Encode::Reset(mfxVideoParam& par)
     return sts;
 }
 
+mfxStatus FEI_Encode::SetRepackCtrl()
+{
+    mfxStatus sts = MFX_ERR_NONE;
+
+    mfxExtFeiHevcRepackCtrl *pRepackCtrl = m_encodeCtrl.GetExtBuffer<mfxExtFeiHevcRepackCtrl>();
+    MSDK_CHECK_POINTER(pRepackCtrl, MFX_ERR_NOT_INITIALIZED);
+
+    sts = m_pFile_repack_ctrl->Read(&(pRepackCtrl->MaxFrameSize), sizeof(mfxU32), 1);
+    MSDK_CHECK_STATUS(sts, "FEI Encode Read repack ctrl MaxFrameSize failed");
+
+    sts = m_pFile_repack_ctrl->Read(&(pRepackCtrl->NumPasses), sizeof(mfxU32), 1);
+    MSDK_CHECK_STATUS(sts, "FEI Encode Read repack ctrl NumPasses failed");
+
+    sts = m_pFile_repack_ctrl->Read(pRepackCtrl->DeltaQP, sizeof(mfxU8), 8);
+    MSDK_CHECK_STATUS(sts, "FEI Encode Read repack ctrl DeltaQP failed");
+
+    if (pRepackCtrl->NumPasses > 8)
+        msdk_printf(MSDK_STRING("WARNING: HEVC NumPasses should be less than or equal to 8\n"));
+
+    return sts;
+}
+
+mfxStatus FEI_Encode::GetRepackStat()
+{
+    mfxStatus sts = MFX_ERR_NONE;
+    mfxI8 repackStatChar[64];
+
+    mfxExtFeiHevcRepackStat *pRepackStat = m_encodeCtrl.GetExtBuffer<mfxExtFeiHevcRepackStat>();
+    MSDK_CHECK_POINTER(pRepackStat, MFX_ERR_NOT_INITIALIZED);
+
+    snprintf(repackStatChar, sizeof(repackStatChar),
+             "%d NumPasses\n", pRepackStat->NumPasses);
+    sts = m_pFile_repack_stat->Write(repackStatChar, strlen(repackStatChar), 1);
+
+    pRepackStat->NumPasses = 0;
```

Comment:
Why should it be zeroed after write operation?

Reply:
It is reset for the next frame.

Reply:
Can we add comment here?